resources/js/components/drawers/LiveMetrics.vue

```diff
@@ -170,10 +214,10 @@ function LiveMetricsToPretty(data: App.Http.Resources.Models.LiveMetricsResource
 		count: count,
 		src: data.url,
 		link: {
-			name: data.photo_id ? "photo" : "album",
+			name: "album",
 			params: {
-				albumid: data.album_id,
```

Comment:
This fixes the links here:
![image](https://github.com/user-attachments/assets/caff8624-0b7f-422c-b6ec-a30f52cb95da)